src/backend/distributed/master/worker_node_manager.c

```diff
@@ -414,6 +430,33 @@ ReferenceTablePlacementNodeList(LOCKMODE lockMode)
 }
 
 
+/*
+ * CitusLocalTablePlacementNodeList returns a single element list containing
+ * the coordinator node if coordinator is added to pg_dist_node, otherwise
+ * errors out.
+ */
+List *
+CitusLocalTablePlacementNodeList()
```

Comment:
Isn't this more  generic than `CitusLocalTablePlacementNodeList `? Like `WorkerNode * CoordinatorNode()`;

Reply:
Makes sense. As after allowing citus local table creation from worker nodes in future, `CitusLocalTablePlacementNodeList` wouldn't be returning the coordinator only.